crates/sql/src/errors.rs

```diff
@@ -0,0 +1,106 @@
+use crate::ast::{ad_hoc, subscription, PrintOperator, SqlSource};
+use derive_more::Display;
+use error_stack::Report;
+use spacetimedb_vm::errors::ErrorVm;
+use sqlparser::ast::{BinaryOperator, Expr, Function, FunctionArg, FunctionArgExpr, Ident, SetExpr, Statement, Value};
+use thiserror::Error;
+
+#[derive(Debug, Display)]
+pub enum Suggestion {
+    #[display(fmt = "Change it to an INNER JOIN")]
+    ChangeJoinToInner,
+    #[display(fmt = "Change to a valid comparison operator: `=, <, >, <=, >=`")]
+    ChangeOpToCmp,
+    #[display(fmt = "Change to several queries")]
+    ChangeOrToManyQueries,
+}
+
```

Comment:
It could be good idea to brainstorm which suggestion could enhance the experience of errors.